src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -848,6 +851,85 @@ eventer_ssl_ctx_new(eventer_ssl_orientation_t type,
   return NULL;
 }
 
+#ifndef OPENSSL_NO_NEXTPROTONEG
+static int next_proto_cb(SSL *ssl, const unsigned char **data,
+                         unsigned int *len, void *arg) {
+  (void)ssl;
+  eventer_ssl_ctx_t *ctx = (eventer_ssl_ctx_t *)arg;
+
+  *data = ctx->npn;
+  *len = (unsigned int)(1 + *ctx->npn);
```

Comment:
how do we handle Len overflow.

Reply:
I don't think it can overflow. ctx->npn is a uint8_t, so [0, 255] + 1 is [1,256] which fits in *len as an unsigned int.